doc/source/drivers/raster/vrt.rst

```diff
@@ -1741,6 +1741,11 @@ target specific band/s as per (:ref:`gdal_translate`).
 
 The effect of the ``outsize`` option (added in GDAL 3.7) is to set the size of the output, in numbers `pixel,line` or in fraction `pixel%,line%` as per (:ref:`gdal_translate`).
 
+The effect of the ``projwin`` option (added in GDAL 3.8) is to select a subwindow from the source image in georeferenced
+coordinates in the same way as (:ref:`gdal_translate`). The value consists of four numeric values separated by commas, in
+the order 'xmin,ymax,xmax,ymin'.
+
+
```

Comment:
Because support for projwin_srs is missing, maybe it would be good to mention also here that the coordinates are  expressed in the SRS of the dataset. Or maybe it is obvious.

Reply:
I wondered that, but also got the srs in the next PR ready 😃